Yona/Yona/Others/Constants.swift

```diff
@@ -143,6 +143,13 @@ struct YonaConstants {
     struct nsUserDefaultsKeys{
         static let isBlocked = "isBlocked"
         static let screenToDisplay = "screenToDisplay"
+        static let timeBucketTabToDisplay = "timeBucketTabToDisplay"
+    }
+    
+    struct timeBucketTabNames{
```

Comment:
I would say this is an enum because there are only limited number of timebucketTabName types, right? Change it to Enum